ctrd/image.go

```diff
@@ -78,25 +78,33 @@ func (c *Client) ListImages(ctx context.Context, filter ...string) ([]types.Imag
 		digest := descriptor.Digest
 
 		size, err := image.Size(ctx, c.client.ContentStore(), platforms.Default())
+		// occur error, skip it
 		if err != nil {
-			return nil, err
+			logrus.Errorf("Get image %s size failed: %v", image.Name, err)
```

Comment:
`("failed to get image size of %s: %v, image.Name, err) `
How about that and keep consistent in the following file? @HusterWan